_i18n/it/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Un firmware open source per tastiere basato su AVR e ARM"
+permalink: /it/
+lang: "it"
+---
+Il firmware QMK è un firmware per tastiera basato su [tmk\_keyboard](http://github.com/tmk/tmk_keyboard) con alcune caratteristiche utili per i controller Atmel AVR e ARM, e più specificamente, la [linea di prodotti OLKB](http://olkb.com), la tastiera [ErgoDox EZ](http://www.ergodox-ez.com), e la [linea di prodotti Clueboard](http://clueboard.co/).
+
+## Tastiere basate su QMK
+
+* [Planck](https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/)
+* [Preonic](https://github.com/qmk/qmk_firmware/blob/master/keyboards/preonic/)
+* [ErgoDox EZ](https://github.com/qmk/qmk_firmware/blob/master/keyboards/ergodox_ez/)
+* [Clueboard](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/)
+* [Cluepad](https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/17/)
+* [Gergo](https://qmk.fm/keyboards/gergo/)
+* [Georgi](https://qmk.fm/keyboards/georgi/)
+* [Atreus](https://github.com/qmk/qmk_firmware/blob/master/keyboards/atreus/)
+
+Questi progetti usano QMK come firmware ufficiale e per default - il tuo e ' scomparso dalla lista? [Facci sapere!](https://github.com/qmk/qmk.fm/issues/new)
+
+Ora QMK è compatibile con [molte altre tastiere e progetti mantenuti dalla comunità](/it/keyboards/).
```

Comment:
This should stay linked to the auto-generated English page until we figure out a way to generate this page in multiple languages.

```suggestion
Ora QMK è compatibile con [molte altre tastiere e progetti mantenuti dalla comunità](/keyboards/).
```